frontend/src/UpdateBook.tsx

```diff
@@ -57,6 +57,17 @@ function UpdateBook(props: UpdateBookProps) {
         >
 
             <form onSubmit={handleEditBook}>
+                <br/>
+                <label>
```

Comment:
Close the label tag before your input and bind it to the input tag with attribute HtmlFor="idOfInput"